### DISCUSSION TITLE
[Feature Request] Calendar

### DISCUSSION THREAD

**Discussion by PunnyBoi** [standardnotes/forum#2485] [Suggestions]
Is it possible to add a small calendar in the app? I mean a calendar in which we can mark special or important events.

**Comment by atmoio** [comment]
Probably as an editor extension, at some point. No real plans at this point though.

**Comment by tej** [comment]
Bump, Highly underrated request.
Need something similar to a Calendar extension or Agenda/Time-based to-do list

it would be nice if I could import external calendar feeds like Google Calendar or Outlook Calendar, but I'm assuming that would cause security issues or something like that so idk.

**Comment by aryanrajseo** [comment]
i am also looking for a small calender like noteplan app. it will help me to write to-do, reminder, checklist, work in specific dates.
